### ISSUE TITLE
Popovers not accessible by screen readers

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#4560]
Popovers like this 

<img width="523" height="312" alt="Image" src="https://github.com/user-attachments/assets/58fbe9d7-6dd6-4bd8-ac55-05f7d3dff4c6" />

currently do not open when tab focus is on their element, likely why screen readers never see them. Bootstrap must have a solution that they use for tooltips, which respond to hover by default and thus also respond to hover-like triggers like tab focus and screen reader focus (this may or may not be the same thing).

**Comment by Bubballoo3** [comment]
I think the css marker we want to use here is `:focus_visible`

**Comment by Bubballoo3** [comment]
It turns out bootstrap supports this with `trigger: 'hover focus'`. This could be easily substituted everywhere we use `trigger: 'hover'`, but #4951 might make us trigger popovers manually, so this should be taken care of with that.
